test/integration/replies_test.rb

```diff
@@ -1,13 +1,94 @@
 require "test_helper"
 
 describe "返信 Integration" do
+  describe "きづき詳細ページ内" do
+    before do
+      @user = login
+      @notice = create(:notice)
+      @base_id = "#{@notice.id}"
+    end
+
+    describe "新規作成" do
+      context "きづき詳細ページを訪れたとき" do
+        before { visit notice_path(@notice) }
+
+        context "正しい値を入力して「返信」したとき" do
+          before do
+            @reply = build(:reply)
+            fill_in "reply_body_#{@base_id}", with: @reply.body
+            find(:css, '#new_reply').click_button "返信する"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.